### ISSUE TITLE
Help on regular sign in

### ISSUE THREAD

**Issue by Ontokrat** [dburles/meteor-two-factor#6]
Hi again @dburles, 

You helped me a month ago with a compatibility issue with `aldeed:collection2-core`: thanks.

Sorry for this newbie question (not an issue...), but I can't understand how on server side, allowing regular sign in.
I can't figure how to define the `twoFactorEnabled` property. And how to set my options?

```
// Optional
// Conditionally allow regular or two-factor sign in
twoFactor.validateLoginAttempt = options => {
  return !! options.user.twoFactorEnabled;
};
```

My idea is to allow regular login for the users who don't have a phone number (`profile.phone`): cause in my App, the code is sended by sms.
Can you help me?
Thanks in advance for reading me so far.

**Comment by dburles** [comment]
Hey @Ontokrat just define it some place on the server. In your case it might look something like:
```js
twoFactor.validateLoginAttempt = options => {
  // Disable two-factor authentication for users without a mobile
  return ! options.user.mobile;
};
```

**Comment by dburles** [comment]
To make it clear `options.user` is the user document for the user signing in.

**Comment by Ontokrat** [comment]
@dburles Sorry, but I dont understand 

> the user document for the user signing in

The field is `profile.phone`

**Comment by dburles** [comment]
See here in the docs how the function behaves https://docs.meteor.com/api/accounts-multi.html#AccountsServer-validateLoginAttempt

So in your case it would be `options.user.profile.phone`.

**Comment by Ontokrat** [comment]
@dburles I'm still stuck in the 2FA way, even if profile.phone is unset... Got no error on the server.. I've got the impression that the function is not called/readed...

**Comment by dburles** [comment]
Clone this repo and try to reproduce the issue and I'll take a look

https://github.com/dburles/two-factor-example

**Comment by Ontokrat** [comment]
@dburles Done the try, on `two-factor-example.js`, added on server-side (line 62) :

```
twoFactor.validateLoginAttempt = options => {
    // Disable two-factor authentication for users without a mobile
    return ! options.user.profile.phone;
  };
```

By default, no user has a `profile.phone` : every time, the 2FA sign in is called.

**Comment by dburles** [comment]
sorry my bad it should be *if* they have a phone number so:

```js
twoFactor.validateLoginAttempt = options => {
  // Disable two-factor authentication for users without a mobile
  return !! options.user.profile.phone;
 };
```

send me a link to your fork if it's still an issue

**Comment by Ontokrat** [comment]
@dburles The issue is still here but I managed the things in a different way : if the user don't have a phone, the code is sending by mail, not by phone. Like that, I keep the 2FA login. But anyway, if you can understand why it's not working, I'm still interested.
Thanks for your time.

**Comment by dburles** [comment]
I had completely forgot about #4 which is the problem here, I've just published `1.1.2` to address this issue.

For accounts that don't require 2FA, you will need to simply call the regular `Meteor.loginWithPassword` method, which means you will need to determine whether they require 2FA or not prior to calling either `Meteor.loginWithPassword` OR `twoFactor.getAuthCode`. Hope that makes sense
